src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,26 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+
+    private static int MIN_AGE = 35;
+    private static String REQUIRED_NATIONALITY = "Ukrainian";
+    private static String REGEX = "-";
+    private static int MIN_DURATION_IN_UK = 10;
+    private static int FROM_YEAR_OF_LIVE_IN_UK = 0;
+    private static int TO_YEAR_OF_LIVE_IN_UK = 1;
```

Comment:
UK - United Kingdom
UA - Ukraine
```suggestion
    private static int MIN_DURATION_IN_UA = 10;
    private static int FROM_YEAR_OF_LIVE_IN_UA = 0;
    private static int TO_YEAR_OF_LIVE_IN_UA = 1;
```